amazon-kinesis/1.0/modules/ROOT/pages/amazon-kinesis-connector-studio.adoc

```diff
@@ -0,0 +1,153 @@
+= Use Anypoint Studio to Configure Amazon Kinesis Data Streams Connector 1.0 - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>
+. <<configure-other-fields,Configure the other connector fields>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Anypoint Connector for Amazon Kinesis Data Streams Connector (Amazon Kinesis Data Streams Connector) to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `kinesis` in the search field.
+. Click *Amazon Kinesis Data Streams Connector* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+=== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure one of these sources to use with Amazon Kinesis Data Streams Connector:
+
+* *Listener*
++
+Enables you to subscribe to a particular data stream. This source uses the Kinesis Client Library (KCL) to listen from a specified initial point in the data stream. It returns dynamically sized batches of records, however, the max size can also be configured.
++
+* *HTTP > Listener*
++
+Initiates a flow each time it receives a request on the configured host and port
++
+* *Scheduler*
++
+Initiates a flow when a time-based condition is met
+
+For example, to configure the *Listener* source, follow these steps:
```

Comment:
```suggestion
For example, to configure the Amazon Kinesis *Listener* source, follow these steps:
```

Reply:
I know that you said that they renamed it to only **Listener**, but in this line you can previously add the connector name just to differentiate to HTTP, and you do not need to bold it, just bold **_Listener._**